content/02.main-text.md

```diff
@@ -109,9 +109,24 @@ The notebook to generate this figure can be [interactively launched using Binder
 ## Manubot features
 
 Manubot is a system for writing scholarly manuscripts via GitHub that is built upon our [Python package](https://github.com/manubot/manubot) of the same name.
-With Manubot, manuscripts are written as plain-text Markdown files, which is well suited for version control using git.
-The Markdown standard itself provides limited yet crucial formatting syntax, including the ability to embed images and format text via bold, italics, hyperlinks, headers, inline code, codeblocks, blockquotes, and numbered or bulleted lists.
+
+TODO: content directory with markdown, metadata, image files.
+
+### Markdown
+
+With Manubot, manuscripts are written as plain-text Markdown files.
+The [Markdown standard](https://spec.commonmark.org/0.28/) itself provides limited yet crucial formatting syntax, including the ability to embed images and format text via bold, italics, hyperlinks, headers, inline code, codeblocks, blockquotes, and numbered or bulleted lists.
 In addition, Manubot relies on extensions from [Pandoc Markdown](https://pandoc.org/MANUAL.html#pandocs-markdown) to enable citations, tables, captions, and equations specified using the popular TeX math syntax.
+Markdown with Pandoc extensions supports most formatting options required for scholarly writing [@doi:10.7717/peerj-cs.112], but currently lacks the ability to cross-reference and automatically number figures, tables, and equations.
+For this functionality, Manubot relies on a the [`pandoc-xnos`](https://github.com/tomduck/pandoc-xnos) suite of Pandoc filters.
+A list of formatting options officially supported by Manubot, at the time of writing, is viewable as [raw markdown](https://gitlab.com/manubot/rootstock/blob/fea9eb80af4ca9d5a116f6c28a6b740272e3c0b9/content/02.delete-me.md#L) and the corresponding [rendered HTML](https://manubot.github.io/rootstock/v/fea9eb80af4ca9d5a116f6c28a6b740272e3c0b9/).
```

Comment:
Note that I link to the GitLab mirror due to https://github.com/isaacs/github/issues/297#issuecomment-478785501 (GitHub does not have a source-code view for markdown files).

Reply:
The choice is between:

+ https://gitlab.com/manubot/rootstock/blob/fea9eb80af4ca9d5a116f6c28a6b740272e3c0b9/content/02.delete-me.md#L
+ https://github.com/manubot/rootstock/raw/fea9eb80af4ca9d5a116f6c28a6b740272e3c0b9/content/02.delete-me.md

The issue with with the raw.githubusercontent.com URL is that you loose the repository context. However, it may be confusing for people to see the GitLab context. I'm now leaning towards switching to the github raw view.

Reply:
There isn't a perfect solution, and I agree that there is context lost with the raw URL.  I also slightly prefer https://github.com/manubot/rootstock/raw/fea9eb80af4ca9d5a116f6c28a6b740272e3c0b9/content/02.delete-me.md